lib/matplotlib/animation.py

```diff
@@ -1371,6 +1374,21 @@ def _repr_html_(self):
         elif fmt == 'jshtml':
             return self.to_jshtml()
 
+    def pause(self):
+        """Pause the animation."""
+        if not self._paused:
+            self._paused = True
+            self.event_source.stop()
+            self._fig.canvas.draw_idle()
+            self._fig.set_animated(False)
```

Comment:
We need to go through and toggle `set_animated` on each of the animated artists, not the whole figure.  The list of blitted figures is stored in `self._drawn_artists`.